taxonomies/origins.txt

```diff
@@ -122,6 +123,24 @@ en:Sicily
 fr:Sicile
 it:Sicilia
 
+# Spanish regions
+
+<en:spain
+es:Andalucía
```

Comment:
for origins, please always set an en: entry as the main entry (as it's done for French regions already).
If there isn't an English "translation", just use whatever the wikipedia page is called. e.g. https://en.wikipedia.org/wiki/Castilla%E2%80%93La_Mancha